MultiPoolMiner.ps1

```diff
@@ -365,10 +365,12 @@ while (-not $API.Stop) {
         $AllPools | Select-Object | ForEach-Object { $_.Price = 0 }
         $AllDevices = @(Get-Device -DevicePciOrderMapping $Config.DevicePciOrderMapping -Refresh | Select-Object)
         if ($API) { $API.AllDevices = $AllDevices } #Give API access to the device information
+        #Load information about the devices, refresh might be required because $AllDevices is cached
```

Comment:
It's already refreshed: 

https://github.com/MultiPoolMiner/MultiPoolMiner/blob/e4a906a1ed55116c8605df49a2ae5b73e3c1696d/MultiPoolMiner.ps1#L366

Reply:
We need -refresh again on the call for normal $Devices (otherwise it will return the cached AllDevices)

Reply:
That should not happen. I'll check the `Get-Device` function shortly.